prusti-interface/src/environment/mir_utils/place_as_local.rs

```diff
@@ -0,0 +1,16 @@
+use rustc_middle::mir;
+
+pub trait PlaceAsLocal {
+    /// Tries to interpret the given `mir::Place` as a local variable. If this fails, it returns `None`.
+    fn as_local(&self) -> Option<mir::Local>;
```

Comment:
How is this different from [`as_local`](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/mir/struct.Place.html#method.as_local)?

Reply:
Ah thanks, the previous compiler version didn't have `as_local`.